q2cli/cli.py

```diff
@@ -0,0 +1,104 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016--, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import click
+import qiime
+from qiime.sdk import PluginManager, SubprocessExecutor
+
+from . import __version__ as q2cli_version
+
+@click.group()
+@click.pass_context
+def cli(ctx):
+    ctx.obj['plugin_manager'] = PluginManager()
```

Comment:
`PluginManager` is instantiated twice, once here and also on line 43. Is that necessary?